livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -39,13 +39,23 @@ class SpeechEvent:
     alternatives: List[SpeechData] = field(default_factory=list)
 
 
+@dataclass
+class STTCapabilities:
+    streaming_supported: bool
```

Comment:
```suggestion
    streaming: bool
```
removing _supported since everything in capabilities are supported